Node/samples/app.js

```diff
@@ -210,6 +212,22 @@ bot.dialog('MentionTeam', function (session) {
     session.send(msg);
     session.endDialog();
 });
+bot.dialog('NotificationFeed', function (session) {
+    // user name/user id
+    var msg = new teams.TeamsMessage(session).text("This is a test notification message.");
+    // This is a dictionary which could be merged with other properties
+    var alertFlag = teams.TeamsMessage.AlertFlag();
+    var notification = msg.sourceEvent({
+        '*': alertFlag
+    });
+    // this should trigger an alert
+    session.send(notification, function (err, response) {
```

Comment:
what is the callback for? session.send() doesn't take a callback. sendBatch() does

Reply:
Sure I will remove this